doc/sphinx-guides/source/admin/timers.rst

```diff
@@ -59,14 +59,37 @@ This job is automatically scheduled to run once a week at 12:30AM local time on
 
 * Create or edit dataverse/src/main/webapp/WEB-INF/ejb-jar.xml, following the :download:`sample file <../_static/admin/ejb-jar.xml>` provided.
 
-* Edit the parameters in the <schedule> section ejb-jar file in the WEB-INF directory to suit your preferred schedule
+* Edit the parameters in the <schedule> section of the ejb-jar file in the WEB-INF directory to suit your preferred schedule
 
   * The provided parameters in the sample file are <minute>, <hour>, and <dayOfWeek>; additional parameters are available
 
     * For a complete reference for calendar expressions that can be used to schedule Timer services see: https://docs.oracle.com/javaee/7/tutorial/ejb-basicexamples004.htm
 
 * Build and deploy the application
 
+* Alternatively, you can insert an ejb-jar.xml file into a provided dataverse war file without building the application.
+
+    * Check if there is already an ejb-jar.xml file in the war file (as I'm writing, the dataverse war file is named dataverse-4.20.war)
```

Comment:
Since this will be for the Dataverse 5 release, we should change this to that - I think you can also jsut say e.g. instead of "as I'm writing"

Reply:
like 'Check if there is already an ejb-jar.xml file in the war file, e.g dataverse-5.war'   and same wherever referenced in the commands?